aip/general/0136.md

```diff
@@ -44,6 +44,9 @@ services. The bullets below apply in all three cases.
   - The name **must not** contain prepositions ("for", "with", etc.).
   - The verb in the name **should not** contain any of the standard method verbs ([Get][],
     [List][], [Create][], [Update][], [Delete][]).
+  - The name **must not** start or end with "async". The verb `Begin` **may**
```

Comment:
Would it be simpler to disallow the use of `Async` anywhere in the RPC name? For example,

> The name **must not** contain the term `Async`.

`rpc GetAsyncJob` -> `function GetAsyncJobAsync` is kind of weird.

Reply:
True... I wanted to avoid being too "blanket" but I can't think of a good example...